site/docs/reference/Connectors/materialization-connectors/Rockset.md

```diff
@@ -55,6 +55,22 @@ materializations:
       	collection: ${table_name}
     source: ${tenant}/${source_collection}
 ```
+
+## Delta updates and reduction strategies
+
+The Rockset connector operates only in [delta updates](../../../concepts/materialization.md#delta-updates) mode.
+This means that Rockset merges documents by key when they arrive, using semantics that are somewhat different than Flow's.
+(FOR REVIEW: include a brief description of the difference here? I don't quite understand the distinction but seems worthwhile to cover in brief)
+
+In most cases, this difference won't affect your data. However, using certain
```

Comment:
> In most cases, this difference won't affect your data

This isn't going to age well, as I expect that reduction annotations will become much more commonly used than they are now. I'm a little torn about whether to leave it for now, but just FYI